src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dto/GeneralPartnerDataDto.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import java.time.LocalDate;
+
+import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.LONG_MAX_SIZE;
+import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.MIN_SIZE;
+import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.REG_EXP_FOR_INVALID_CHARACTERS;
+import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.SHORT_MAX_SIZE;
+
+public class GeneralPartnerDataDto {
+
+    public static final String NAME_MIN_SIZE_MESSAGE = "general partner name must be greater than {min}";
```

Comment:
You probably need to check, but I think we need separate messages for each field, which should reference the field labels displayed in the web page, so 'First name', 'Last name' and 'Previous name'. These messages will then end up being displayed on the web page, when returned by the API.

Reply:
You have a point, but let's address comments regarding status 400 errors in a separate PR.

Reply:
👍 will resolve.